spec/units/peep_spec.rb

```diff
@@ -0,0 +1,14 @@
+require './lib/peep'
+
+RSpec.describe Peep do 
```

Comment:
Great to see thorough user tests. A lot of people forget them so glad to see you are thoroughly testing the classes in isolation